test/api/v01/unitary_test.rb

```diff
@@ -39,24 +39,44 @@ def test_should_geocode_from_fields
 
   def test_should_not_geocode_without_country
     get '/0.1/geocode', {api_key: 'demo', query: 'Place Pey Berland, Bordeaux'}
-    assert !last_response.ok?, last_response.body
+    assert last_response.status, 400
```

Comment:
this `assert` is already true...
correct line: `assert_equal 400, last_response.status`